src/main/resources/assets/hyperium/sounds.json

```diff
@@ -1,2079 +0,0 @@
-{
```

Comment:
Are you sure this is unnecessary? I recall issues with sounds in the past. Have you built the client and installed it from the built jar to test it?

Reply:
JSONLint isn’t happy about it.

Reply:
tested anything i can, didnt break at all